api/v1alpha3/azuremachine_types.go

```diff
@@ -51,12 +51,22 @@ type AzureMachineSpec struct {
 	Image *Image `json:"image,omitempty"`
 
 	// Identity is the type of identity used for the virtual machine.
-	// The type 'SystemAssigned' is an implicitly created identity
-	// The generated identity will be assigned a Subscription contributor role
+	// The type 'SystemAssigned' is an implicitly created identity.
+	// The generated identity will be assigned a Subscription contributor role.
+	// The type 'UserAssigned' is a standalone Azure resource provided by the user
+	// and assigned to the VM
 	// +kubebuilder:default=None
 	// +optional
 	Identity VMIdentity `json:"identity,omitempty"`
 
+	// UserAssignedIdentities is a list of standalone Azure identities provided by the user
+	// The lifecycle of a user-assigned identity is managed separately from the lifecycle of
+	// the AzureMachine.
+	// The format of an identity is:
+	// 'azure:////subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'
```

Comment:
this comment is more appropriate for providerID no?

Reply:
yeah, will move it

Reply:
```suggestion
```

Reply:
looks like I missed this file in the commit, fixed now